test/colony-network-recovery.js

```diff
@@ -250,28 +239,16 @@ contract("Colony Network Recovery", accounts => {
           await colony.mintTokens(1000000000000000);
           await colony.bootstrapColony([accounts[0]], [1000000000000000]);
 
-          let addr = await colonyNetwork.getReputationMiningCycle.call(false);
-          let repCycle = await IReputationMiningCycle.at(addr);
-
-          await miningClient.addLogContentsToReputationTree();
-          await forwardTime(MINING_CYCLE_DURATION, this);
-          await miningClient.submitRootHash();
-
-          addr = await colonyNetwork.getReputationMiningCycle.call(true);
-          repCycle = await IReputationMiningCycle.at(addr);
-          await repCycle.confirmNewHash(0);
-
           await miningClient.addLogContentsToReputationTree();
-          await forwardTime(MINING_CYCLE_DURATION, this);
-          await miningClient.submitRootHash();
-
-          addr = await colonyNetwork.getReputationMiningCycle.call(true);
-          repCycle = await IReputationMiningCycle.at(addr);
+          await advanceMiningCycleNoContest(colonyNetwork, this, miningClient);
 
+          const addr = await colonyNetwork.getReputationMiningCycle(true);
+          const repCycle = await IReputationMiningCycle.at(addr);
```

Comment:
Use the new `getActiveMineCycle` helper?

Reply:
Ok updated

Reply:
There are a still a few places where a line or two could be saved with this pattern (e.g. https://github.com/JoinColony/colonyNetwork/pull/441/files#diff-15aaaaa81a55d904a938772d3e2a3a82R387) - basically, most (though not all) places where `getReputationMiningCycle(true)` still exists.

Reply:
The only kicker is that we occasionally use `addr` in comparisons, but we could always drop in `repCycle.address`.

Reply:
Alright, try this on for size